src/core/server/saved_objects/migrations/actions/integration_tests/actions.test.ts

```diff
@@ -164,14 +164,14 @@ describe('migration actions', () => {
         indices: ['existing_index_with_docs'],
       });
       await expect(task()).resolves.toMatchInlineSnapshot(`
-              Object {
-                "_tag": "Left",
-                "left": Object {
-                  "message": "[unsupported_cluster_routing_allocation] The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue.",
-                  "type": "unsupported_cluster_routing_allocation",
-                },
-              }
-            `);
+        Object {
+          "_tag": "Left",
+          "left": Object {
+            "message": "[unsupported_cluster_routing_allocation] The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue.",
+            "type": "unsupported_cluster_routing_allocation",
```

Comment:
Aside: Should we change the error type to `incompatible_cluster_routing_allocation`? `incompatible` feels more appropriate than `unsupported`.

Reply:
yeah that makes sense, but I'd prefer to leave it out of this bug fix.